src/main/java/frc/robot/commands/vision/TestVisionDistance.java

```diff
@@ -28,11 +32,13 @@ public class TestVisionDistance extends CommandBase{
   // How many seconds between each data print
   private static final double kPrintDelay = 0.25;
 
-  public TestVisionDistance(double speed, Drivetrain drive, Vision vision){
+  public TestVisionDistance(double speed, Drivetrain drive, Vision vision, ShuffleboardTab visionTab){
     addRequirements(drive);
     m_drive = drive;
     m_speed = speed;
     m_vision = vision;
+    m_outputData = new double[4];
+    visionTab.addDoubleArray("Distance Test Results", ()->getOutputData());
```

Comment:
again please label what you outputing rather than just outputing an array and assuming people know what and the order of the output